graphistry/__init__.py

```diff
@@ -15,7 +15,7 @@
     description,
     bind,
     style,
-    addStyle,
+    add_style,
```

Comment:
same, add cleaner for now, don't needlessly break users (yet)

Reply:
maybe we should revert back to addStyle

Reply:
we changed this for a doc issue we were facing